test/unit/test_bigquery_adapter.py

```diff
@@ -931,3 +941,16 @@ def test_convert_time_type(self):
         expected = ['time', 'time', 'time']
         for col_idx, expect in enumerate(expected):
             assert BigQueryAdapter.convert_time_type(agate_table, col_idx) == expect
+
+
+@pytest.mark.parametrize(
```

Comment:
We don't seem to use this style of test very often, but I actually really like it so I'm happy to introduce it to this file too.